radixdlt-java/src/main/java/com/radixdlt/client/application/translate/ParticleReducer.java

```diff
@@ -1,14 +1,19 @@
 package com.radixdlt.client.application.translate;
 
-import com.radixdlt.client.core.ledger.TransitionedParticle;
+import com.radixdlt.client.core.atoms.particles.Particle;
 
 /**
- * Java version of redux-like reducer where particles are the actions
+ * Temporary class for reducing up particles into an application state.
+ * TODO: Introduce the AtomOS layer in between Application and Particles
+ * TODO: which will reduce fungible, indexable, and payload particle types
+ * TODO: into a state machine upon which application state can be built on
+ * TODO: top of
```

Comment:
God bless for this comment!
I love when a developer cares enough for his/her audience to give a few lines of context where needed.

Reply:
+1
Please let's not call it 'payload' as discussed